testing/trino-testing/src/main/java/io/trino/testing/BaseConnectorTest.java

```diff
@@ -1718,6 +1718,41 @@ public void testAddColumn()
         assertFalse(getQueryRunner().tableExists(getSession(), tableName));
     }
 
+    @Test
+    public void testAddColumnWithComment()
+    {
+        if (!hasBehavior(SUPPORTS_ADD_COLUMN)) {
+            assertQueryFails("ALTER TABLE nation ADD COLUMN test_add_column_with_comment bigint", "This connector does not support adding columns");
+            return;
+        }
+
+        skipTestUnless(hasBehavior(SUPPORTS_CREATE_TABLE));
+
+        String tableName;
+        try (TestTable table = new TestTable(getQueryRunner()::execute, "test_add_column_with_comment", "(_bigint bigint COMMENT 'test column comment')")) {
```

Comment:
nit: `a_bigint` - it's possible some databases don't allow identifiers to begin with `_`.